bin/oscapd-cli

```diff
@@ -30,6 +30,78 @@ import sys
 from datetime import datetime
 import os.path
 
+class TaskAccessor:
```

Comment:
This is an old-style class, make sure you inherit `object` to use the new-style classes in python2.